src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Main.cs

```diff
@@ -161,6 +172,12 @@ public List<Result> Query(Query query)
             return results;
         }
 
+        // This function uses the Windows indexer and returns the list of results obtained. This version is required to implement the interface
+        public List<Result> Query(Query query)
+        {
+            return Query(query, false);
+        }
+

```

Comment:
I think it would make things simpler if we just modify the query function in the IPlugin interface and make the second parameter have a default value of false (like how the QueryForPlugin was modified). That way we don't have to have this function just to implement the interface and also we wouldn't need another interface just to add a second argument for the Query function.

Reply:
I had tried this but it requires updating all the plugins, i.e. adding the second parameter for all the plugins. I wanted to make a default implementation in the interface such that the two param version calls the single param version by default, but this is supported only in C# 8 (we currently use 7.3). Updating all the plugins also means that any existing Wox plugins would also need to be modified in order to be used with PT Run.